cpp/arcticdb/storage/lmdb/lmdb_storage.hpp

```diff
@@ -46,11 +46,18 @@ class LmdbStorage final : public Storage {
 
     inline bool do_fast_delete() final;
 
+    void close() override;
+
     void do_iterate_type(KeyType key_type, const IterateTypeVisitor& visitor, const std::string &prefix) final;
 
     bool do_key_exists(const VariantKey & key) final;
 
-    ::lmdb::env& env() { return *env_;  }
+    ::lmdb::env& env() {
+        if (!env_) {
+            raise<ErrorCode::E_UNEXPECTED_LMDB_ERROR>("Unexpected LMDB Error: Invalid operation: LMDB environment has been removed");
```

Comment:
Worth explaining that the root cause might be that the library has been deleted?